### PR TITLE
Add support for ethtool coalesce settings

### PR DESCRIPTION
**Pull request by ichigo663**
Also add related tests.

Resolves: #112

### CODE REVIEW

**Comment by ichigo663** [comment]
Supersedes #234

File: tests/playbooks/tests_ethtool_coalesce.yml [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+              - name: "{{ interface }}"
+                type: ethernet
+                autoconnect: no
+                ip:
+                  dhcp4: no
+                  auto6: no
+        - name: Get profile's coalescing options
+          command: >
+              grep -q ^ETHTOOL_OPTS
+              /etc/sysconfig/network-scripts/ifcfg-{{ interface }}
```
Comment (cathay4t) [inline tests/playbooks/tests_ethtool_coalesce.yml:44]: Fedora 33 is using key-file by default now. You test should not use hard coded path.
In stead of checking what is written in profile, can you find a way to check whether certain coalesce option is applyed to kernel.

**Review by cathay4t — CHANGES_REQUESTED** [review]

File: tests/playbooks/tests_ethtool_coalesce.yml (hunk 1 above)
Comment (ichigo663) [reply tests/playbooks/tests_ethtool_coalesce.yml:44]: I'm afraid not, coalesce settings need kernel support and very few drivers have it. One option could be to use netdevsim but coalesce settings will be supported from Linux 5.11 onwards.

That's why I'm checking the profile instead.

Maybe telling NM to use ifcfg for this test is an option, what do you think?

Comment (cathay4t) [reply tests/playbooks/tests_ethtool_coalesce.yml:44]: How about use `nmcli -f ethtool  c  show  eth1`?

Comment (ichigo663) [reply tests/playbooks/tests_ethtool_coalesce.yml]: So simple that I did not think about it, I changed the PR to use `nmcli -g ethtool`.
PR updated, thanks!

**Review by cathay4t — APPROVED** [review]
LGTM. Thanks.

**Pull request merged**